sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner_test.py

```diff
@@ -377,11 +377,6 @@ def process_timer(
       assert_that(actual, equal_to(expected))
 
   def test_pardo_timers_clear(self):
-    if type(self).__name__ != 'FlinkRunnerTest':
```

Comment:
As mentioned in the comment, `test_pardo_timers` and `test_pardo_timers_clear` could be merged. I had to fork it at that time to not lose the coverage for `fn_api_runner`.

Reply:
I think it would be cleaner to add an override in the subclasses than test on `__name__` here.

Reply:
We have problems if overriding subclass. The `PortableRunnerTest` doesn't support clearing timer, but its subclass `FlinkRunnerTest` and `SparkRunnerTest` support it.

Reply:
They are passing now. It seems like because of my FnApiRunner changes.